.github/workflows/test.yaml

```diff
@@ -23,22 +23,16 @@ jobs:
       fail-fast: false
       matrix:
         os: ['ubuntu-latest']
-        python-version: ['3.7']
+        python-version: ['3.7', '3.8', '3.9', '3.8', '3.9', '3.10']
```

Comment:
`['3.7', '3.8', '3.9', '3.8', '3.9', '3.10']`

3.8 and 3.9 are specified twice

Reply:
good catch, no need to redundantly test them